proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/Const.java

```diff
@@ -188,10 +188,14 @@
         dataStreamToModel.put("RAW", BinaryEditor.RAW_ID);
         dataStreamToModel.put("MC_IMGGRP", BinaryEditor.NDK_ARCHIVAL_ID);
         dataStreamToModel.put("ALTOGRP", "ALTO");
+        dataStreamToModel.put("UC_IMGGRP", "NDK_USER");
```

Comment:
Tohle by mělo být definováno jako konstanty. Hodnoty jsou použité na spoustě míst.